### DISCUSSION TITLE
Servo Shutter feature request is to set pwm to 0 after movement is finished.

### DISCUSSION THREAD

**Discussion by mikep1998** [arendst/Tasmota#10873] [Q&A] [answered]
With servo controlled shutters the shutter code current leaves the PWM valve for the servo motor at the target position.  This leaves the servo rotation in a locked position. The servo can not be moved by hand until the power to the servo is removed or the PWM is set to 0.

I do not have a relay for the servo power. 

I would like an option to allow the PWM valve set to 0 after the movement is finished to allow the rotation of the blinds servo by hand. 

This also reduces the power consumption of the servo motor.

**Comment by TrDA-hab** [comment, marked as answer]
Did you see it?
https://github.com/arendst/Tasmota/discussions/10443

**Reply by mikep1998** [reply]
Yes that is what I used to set it up.

**Comment by mikep1998** [comment]
This issue is now resolved I put a #define to set PWM to 0 in the ShutterPowerOff routine.
